tests/contest/runtimetest/src/tests.rs

```diff
@@ -332,6 +334,29 @@ pub fn validate_sysctl(spec: &Spec) {
     }
 }
 
+pub fn validate_linux_mount_label(spec: &Spec) {
```

Comment:
@YJDoc2 @utam0k 

I would like to clarify these points.
1. Should I implement `validate_linux_mount_label` ?
2. If so, how should I get actual mount label?
    a) Is there any useful tools I can use for getting mount label? I couldn't use `findmnt`. I am using proc/self/mountinfo instead, but this file doesn't inlcude mount label itself.
    b) Or should I implement `test_inside_container` and `prepare_bundle` function in `linux_mount_label_test.rs` by myself to get `rootfs` value and pass root value to `validate_linux_mount_label`?

Reply:
AFAI, there isn't a good method to validate it, so I think we can't test it. I have checked with runc and crun, but they don't test it.

Reply:
@YJDoc2 Do you have any good idea?